### DISCUSSION TITLE
2025 Academic Conference Books

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#435] [General]
2024년 6월 28일 강릉 워크샵에서
2025년 읽을 책을 선정했습니다.

- #428 

올해는 워크샵 참석 인원이 2년 전과 같이 세 명이어서
태형님 업서님이 한 권씩 골랐고 저도 하나 골랐습니다.

## 2025 Academic Conference Books

| 태형 | 업서 | 종필 |
|-----|-----|-----|
| [![도메인 주도 설계란 무엇인가?](https://image.aladin.co.kr/product/1262/3/cover500/8966260128_1.jpg)](http://aladin.kr/p/vFpYb) | [![타입으로 견고하게 다형성으로 유연하게](https://image.aladin.co.kr/product/32634/2/cover500/8966264174_1.jpg)](http://aladin.kr/p/c4Zpt) | [![더 나은 프로그래머 되는 법](https://image.aladin.co.kr/product/33837/63/cover500/k192930998_1.jpg)](http://aladin.kr/p/yqSGA) |

## 2025 Academic Conference Schedules

- 2024 Home Coming Day 행사가 열리는 12월 14일 토요일 부터 멤버 모집시작, 12월 27일 금요일 까지 모집
- 2024-12-27 금요일 오후 10시, 오리엔테이션 진행 (멤버 소개, 토론 진행 방식 소개, 책 읽는 순서 정하기 등)
- 2024-12-27 부터 2025-06-13 까지, 격주로 금요일 오후 10시에 온라인 미팅 진행, 총 12 sprint (1 sprint == 2 weeks)
- 2025-06-27 금요일, 2025 Academic Conference Workshop 진행, 대부도 방아머리해변(경기도 안산시 단원구) 예정

**Comment by yeslee-v** [comment]
오, 업서님께서 추천하신 책 저자분 강연 듣고 읽어보고 싶었는데 선정되었군요!
